nativelib/src/main/resources/scala-native/scala-native.properties

```diff
@@ -0,0 +1,13 @@
+# output via debugging
+project.groupId = org.scala-native
+project.artifactId = nativelib
+# setup define based on GC selected
+project.gcProject = true
```

Comment:
Is this one really needed? It can be concluded based on groupId/artifactId 
I guess that one these 2 would be sufficient

Reply:
This was just so there doesn't have to be any special logic run on the implementing side - this is an internal setting so we can just include it only on the proper project. It just simplified things a bunch.